vips/vip-160/vip-160-testnet.ts

```diff
@@ -0,0 +1,158 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const THE = "0xB1cbD28Cb101c87b5F94a14A8EF081EA7F985593";
+const TWT = "0xb99C6B26Fdf3678c6e2aff8466E3625a0e7182f8";
+const POOL_REGISTRY = "0xC85491616Fa949E048F3aAc39fbf5b0703800667";
+const VTOKEN_RECEIVER_THE = "0x1c6C2498854662FDeadbC4F14eA2f30ca305104b";
+const VTOKEN_RECEIVER_TWT = "0x1c6C2498854662FDeadbC4F14eA2f30ca305104b";
+const VTHE_DeFi = "0x5Ec06c9dD9654d42B69A8EdBBa99b1e8Afa0D4C0";
+const VTWT_DeFi = "0x4C94e67d239aD585275Fdd3246Ab82c8a2668564";
+const REWARD_DISTRIBUTOR_THE = "0x5cBf86e076b3F36a85dD73A730a3567FdCA0D21E";
+const DEFI_COMPTROLLER = "0x23a73971A6B9f6580c048B9CB188869B2A2aA2aD";
+const BINANCE_ORACLE = "0xCeA29f1266e880A1482c06eD656cD08C148BaA32";
```

Comment:
This is the Chainlink oracle. It's ok, because only the Chainlink oracle accepts direct price, but I would rename the variable to `CHAINLINK_ORACLE` to avoid any confusion

Reply:
Fixed [dc110f6](https://github.com/VenusProtocol/vips/pull/60/commits/dc110f6f374da94a0436b5ca1d1389f9dfc77d29)